src/build.py

```diff
@@ -1416,20 +1416,40 @@ def ValidateLLVMTorture(indir, ext, opt):
 
 class Build(object):
     def __init__(self, name_, runnable_, os_filter=None, is_default=True,
-                 *args, **kwargs):
+                 incremental_build_dir=None, *args, **kwargs):
         self.name = name_
         self.runnable = runnable_
         self.os_filter = os_filter
-        self.is_deafult = is_default
```

Comment:
This removes the only use of the `is_default` parameter, so perhaps it is not needed?